extensions/dacpac/src/test/wizard.test.ts

```diff
@@ -74,10 +74,6 @@ describe('Dacfx wizard', function (): void {
 		wizard.model.upgradeExisting = true;
 		should.equal(wizard.isSummaryPage(3), true);
 
-		// summary page should be 3 for generate deploy script
```

Comment:
it shouldnt be 3rd?

Reply:
this is getting removed as part of some cleanup because there isn't a generate deploy script operation anymore. Originally, generate deploy script was its own operation, but a year ago I removed that path and made generate script just another wizard button for the deploy to existing db scenario.